### PR TITLE
Fixed this issue 

### PR DESCRIPTION
**Pull request by Bharadwajshivam28**
Fixed the color of button Browse project by giving it White color which sets it best. Also added an arrow at side of Browse project button and made the text inside it bolder to look more attractive

### CODE REVIEW

**Comment by Bharadwajshivam28** [comment]
@priyankarpal I have also gave the documentation section a box which looks more good....

old one...
Screenshot from 2023-04-12 22-11-15

new one...
Screenshot from 2023-04-12 22-04-44

File: README.md
Comment (priyankarpal) [inline README.md:1]: could you share why you removed them?

File: contributing.md
Comment (priyankarpal) [inline contributing.md:1]: also we're using `pnpm` in this project. so please use `pnpm`

File: index.html
Comment (priyankarpal) [inline index.html:1]: We're using the `JetBrains Mono` font as a primary font so don't remove it.

File: package-lock.json
Comment (priyankarpal) [inline package-lock.json:1]: Delete the file

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (Bharadwajshivam28) [reply package-lock.json:1]: > Delete the file

I will do all the changes.....

**Comment by priyankarpal** [comment]
In this PR you changed unnecessary changes so don't do it. 🙏🏻 You can close this PR & create a new one.

Also, update your fork repository.
- we're using `pnpm` 
- `JetBrains Mono` Font
- Also use `TailWind CSS`

**Comment by priyankarpal** [comment]
> > In this PR you changed unnecessary changes so don't do it. 🙏🏻 You can close this PR & create a new one.
> > Also, update your fork repository.
> > 
> > * we're using `pnpm`
> > * `JetBrains Mono` Font
> > * Also use `TailWind CSS`
> 
> Ok sir due to some issue..i deleted the changes and because of that all thing's got mixed up and these happened...
> 
> I will create a new pr by correcting all

ok then I'm closing this PR

**Pull request closed without merging**